AppCenter/AppCenter/Internals/HttpClient/MSHttpClient.h

```diff
@@ -22,4 +22,11 @@
  */
 - (instancetype)initWithMaxHttpConnectionsPerHost:(int)maxHttpConnectionsPerHost;
 
+/**
+ * Enables or disables the client. All pending requests are canceled and discarded upon disabling.
+ *
+ * @param isEnabled The desired enabled state of the client - pass `YES` to enable, `NO` to disable.
+ */
+- (void)setEnabled:(BOOL)isEnabled;
```

Comment:
What's the purpose of enable/disable HttpClient?

Reply:
To actually cancel the http calls while pause just says don't retry but keep calls in flight.
Pause is called on network down. disable is called on module disabled.